src/styles/style.scss

```diff
@@ -0,0 +1,142 @@
+
+$white: #fff;
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
```

Comment:
Mind variable namings

Dont use exact colors, but be informative
for instance:
white -> main-inverse, primary-inverse
blue-accent - moyo-accent, accent-primary